.github/workflows/push.yml

```diff
@@ -3,31 +3,31 @@ name: Entur/Docker/Push
 on:
   workflow_call:
     inputs:
-      image_artifact_name:
+      image_name:
```

Comment:
Actually it is the name of the GitHub artifact, not the image in the artifact. But the description is clear on that, so this is ok for me.

Reply:
Currently it does not seem to use the previously built image. 
I am OK to merge it like this, and add an issue. If we _require_ a build step (like we do now) then we should at least re-use the work done by it :) 

Maybe the meta step needs to be in the build? Not sure, I guess we should check the docs again - I seem to remember they had an example for splitting build and push?

Reply:
It is _also_ the actual name of the image, so it serves double duty. I think it's clearer when we emphasize the aspect that our developers are used to think about; the actual image name 👍🏻

Reply:
The image in the GitHub artifact is 'image_name'.tar.

And it is used in the build step. Ref. https://docs.docker.com/build/ci/github-actions/test-before-push/

Reply:
If I understand the example correct

Reply:
But when looking at the job output I am not sure. https://github.com/entur/gha-docker/actions/runs/8442415988/job/23123788816#step:19:229
🤔